packages/store/package.json

```diff
@@ -16,23 +16,21 @@
   },
   "types": "ts/index.ts",
   "scripts": {
-    "build": "pnpm run build:tightcoder && pnpm run build:codegen && pnpm run build:abi && pnpm run build:typechain && pnpm run build:js",
-    "build:abi": "forge build --out abi --skip test script",
-    "build:codegen": "tsx ./ts/scripts/tablegen.ts",
+    "build": "pnpm run build:mud && pnpm run build:abi && pnpm run build:typechain && pnpm run build:js",
+    "build:abi": "forge build --extra-output-files abi --out abi --skip test script",
     "build:js": "vite build",
-    "build:tightcoder": "tsx ./ts/scripts/codegenTightcoder.ts && prettier --write '**/*.sol'",
+    "build:mud": "tsx ./ts/scripts/tablegen.ts",
     "build:typechain": "typechain --target ethers-v5 'abi/**/*.sol/!(*.abi).json'",
     "clean": "pnpm run clean:abi && pnpm run clean:typechain && pnpm run clean:js",
     "clean:abi": "rimraf abi",
     "clean:js": "rimraf dist",
     "clean:typechain": "rimraf types",
     "docs": "rimraf API && hardhat docgen",
     "gas-report": "../cli/dist/mud.js gas-report --path test/* --path test/**/* --save gas-report.json",
```

Comment:
😭  Still need CLI here, but at least this isn't in the build path. I tried to add CLI as a dep, but that creates circular deps again. I also tried to add CLI as a dep in the root package, but there are no bins to link when it attempts to install from the workspace.

We could use whatever npm version is published, but that seems not ideal if we ever want to test changes to the gas report and want to use the local/workspace version.

We could also move the gas report out to common?

Reply:
the tablegen treatment for gas-report would make sense imo, except yeah it'd go to common rather than store/world. And cli just gets a wrapper over the imported script

Reply:
added an issue to track this here: https://github.com/latticexyz/mud/issues/667